pkg/flowkit/services/tests.go

```diff
@@ -0,0 +1,79 @@
+/*
+ * Flow CLI
+ *
+ * Copyright 2019 Dapper Labs, Inc.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *   http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package services
+
+import (
+	"fmt"
+	"path"
+
+	"github.com/onflow/flow-cli/pkg/flowkit"
+	"github.com/onflow/flow-cli/pkg/flowkit/output"
+
+	"github.com/onflow/cadence/runtime/common"
+	"github.com/onflow/cadence/test-framework"
+)
+
+// Tests is a service that handles all tests-related interactions.
+//
+type Tests struct {
+	logger output.Logger
+}
+
+// NewTests returns a new tests service.
+//
+func NewTests(
+	logger output.Logger,
+) *Tests {
+	return &Tests{
+		logger: logger,
+	}
+}
+
+// Execute test scripts.
+//
+func (t *Tests) Execute(code []byte, scriptPath string, readerWriter flowkit.ReaderWriter) (test_framework.Results, error) {
+	runner := test_framework.NewTestRunner().
+		WithImportResolver(func(location common.Location) (string, error) {
+			stringLocation, isFileImport := location.(common.StringLocation)
```

Comment:
address imports are not possible?

Reply:
Address imports are handled at the cadence/test-framework side. They won't reach here.
This resolver is only used for resolving local file imports.